printer_mods/hartk1213/Voron2.4_SexBolt_ZEndstop/README.md

```diff
@@ -0,0 +1,53 @@
+# SexBolt Z endstop 
+![1](Images/9.png) 
+
+This an alternative way to do the Z endstop pin in a V2, instead of using a deflanged GT2 20T pulley, it uses 2  5x7x8 sleeve copper bearings/bushing 
+this is pretty simple to assemble. The reason behind this is to help keep the pin from falling out.
+
+
+
+##
+
+### BOM
+Screws			
+Size|Qty| Link
```

Comment:
This table displays as one line, please see https://www.markdownguide.org/extended-syntax/#tables for how it should be formatted.

Reply:
To be proper markdown it needs the vertical line at the start and end of every row.

Reply:
Fixed